### PR TITLE
doc(CHANGELOG.md): add entry for v2.0.0-beta4

### PR DESCRIPTION
**Pull request by vdice**
# Summary of Changes

Updates CHANGELOG.md with beta4 changes.
# Pull Request Hygiene TODOs

Please make sure the below checklist is complete.
- [x] Your pull request is concise and to the point (make another PR for refactoring nearby code)
- [x] Your commits are squashed into logical units of work
- [x] Your commits follow the [commit style guidelines](http://docs.deis.io/en/latest/contributing/standards/#commit-style-guide)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,19 @@
+### v2.0.0-beta3 -> v2.0.0-beta4
+
+#### Features
```
Comment (mboersma) [inline CHANGELOG.md]: Can you omit this empty section? Sorry, I'm a broken record.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,19 @@
...
+#### Fixes
+
+ - [`7a29e44`](https://github.com/deis/dockerbuilder/commit/7a29e44d44e12ae6543f0c69eac6121721ac8b01) docker-push: the docker push errors
+
+#### Documentation
+
+ - [`63ba6de`](https://github.com/deis/dockerbuilder/commit/63ba6de33d1cdde4f9d33b6af394b6480887053d) badge: added code-beat badge
+ - [`b52ce7d`](https://github.com/deis/dockerbuilder/commit/b52ce7ddfb07811984a1db1ab558cfa62261c9b8) CHANGELOG.md: update for v2.0.0-beta3
+
+#### Maintenance
```
Comment (mboersma) [inline CHANGELOG.md]: Also this one.

**Comment by arschles** [comment]
LGTM assuming the empty sections are removed

File: CHANGELOG.md (hunk 1 above)
Comment (vdice) [inline CHANGELOG.md]: @mboersma thank you, yes missed the tidying up in this one in my hurry... should be cleaned up now!

**Pull request merged**